specs/interop/supervisor.md

```diff
@@ -280,3 +324,111 @@ Returns: RPC error if the `minSafety` is not met by one or more of the access en
 
 The access-list entries represent messages, and may be incomplete or malformed.
 Malformed access-lists result in an RPC error.
+
+### Errors
+
+The Supervisor RPC API uses standard JSON-RPC error codes along with specific application error codes to provide deterministic error handling across different implementations.
+
+#### Standard JSON-RPC Error Codes
+
+##### `-32700`
+
+- `Parse error`: Invalid JSON
+
+##### `-32600`
+
+- `Invalid request`: This can happen when your request is malformed. Additionally, some of the provides use this code to signal that a particular method is not available or requires switching to a paid tier.
+
+##### `-32601`
+
+- `Method not found`: This usually happens when the method is not available with a given provider. If you need to use a specific method that's not supported by the provider, try changing the provider or consider switching to a more expensive tier.
+
+##### `-32602`
+
+- `Invalid params`: This happens when the request parameters are invalid. To fix, double check the parameters that you pass with the request and make sure they comply with the spec.
+
+##### `-32603`
+
+- `Internal error`: This might happen when the node reverts during the request execution. This can also happen if the request is malformed or invalid.
+
+##### `-32000`
+
+- `Invalid input`: Missing or invalid parameters
+
+##### `-32001`
+
+- `Resource not found`: This usually happens when calling a method that's not supported. Try using a different method or switch providers.
+
+##### `-32002`
+
+- `Resource unavailable`: Requested resource not available
+
+##### `-32003`
+
+- `Transaction rejected`: Transaction creation failed
+
+##### `-32004`
+
+- `Method not supported`: Method is not implemented
+
+##### `-32005`
+
+- `Limit exceeded`: Request exceeds defined limit
+
+##### `-32006`
+
+- `JSON-RPC version not supported`: Version of JSON-RPC protocol is not supported
+
+#### Protocol Specific Error Codes
+
+##### `-32100`
+
+- `unknown chain`: happens when a chain is unknown, not in the dependency set.
```

Comment:
```suggestion
- `unknown chain`: Happens when a chain is unknown, not in the dependency set.
```
let's stick to proper sentences in specs, so starting with a capital letter and ending with full stop